catalog.html

```diff
@@ -24,10 +24,11 @@
           </li>
         </ul>
         <a class="navigation-logo" href="index.html">
-          <img class="page-header-logo" alt="Логотип Седоны" src="images/logos/logo.svg">
+          <img class="page-header-logo" alt="Логотип Седоны" src="images/logos/logo.svg" width="138px" height="76px">
```

Comment:
Атрибуты ширины и высоты пишутся без `px`
```suggestion
          <img class="page-header-logo" alt="Логотип Седоны" src="images/logos/logo.svg" width="138" height="76">
```

Reply:
Во всех местах это надо поправлять, соответственно.
И не забывай, что после изменений в коде нужно проходиться валидатором, он как раз такие ошибки отсекает:
![изображение](https://user-images.githubusercontent.com/37917037/142764437-66db011c-a3a4-4344-a6fd-0e33d2c5c244.png)